### ISSUE TITLE
Chrome 60 without DNT is identified as "to track"

### ISSUE THREAD

**Issue by agross** [Alex-D/Cookies-EU-banner#23]
Chrome 60, I disabled DNT in settings. `this.isToTrack()` returns `undefined` which is then negated to `true`.

```js
isToTrack: function () {
      var dnt = navigator.doNotTrack || navigator.msDoNotTrack || window.doNotTrack; // dnt === undefined
      return (dnt !== null) ? (dnt && dnt !== 'yes' && dnt !== 1 && dnt !== '1') : true; // dnt && dnt !== 'yes' && dnt !== 1 && dnt !== '1' === undefined
    },
```

**Comment by agross** [comment]
I tested a bit more with Chrome and Edge:

In Chrome `dnt === undefined`, hence `(dnt !== null) === true` .

In Edge, `dnt === null`, hence `(dnt !== null) === false`.

**Comment by Alex-D** [comment]
Should be fixed in 1.2.9.

**Comment by agross** [comment]
Thank you!
